src/main/java/com/rtb/projectmanagementtool/project/ProjectTestServlet.java

```diff
@@ -0,0 +1,37 @@
+// Just using this Servlet to test the Project & ProjectController classes.
+// I didn't know any other ways to test the classes other than through unit tests, which I
+// couldn't get working.
+// Feel free to mess around with it and test some of the functionality/see how it can be improved
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/my-project-test")
+public class ProjectTestServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Good idea putting together a test for these.  I would suggest starting with unit tests first though before putting together a servlet.

Also keep in mind servlets are accessible to the public (unless if you setup access restrictions), so you wouldn't want to have test logic exposed publicly.

Finally It would be best to run tests off of a test Datastore instance.  This way you won't create test data into your prod database.